docs/source/conf.py

```diff
@@ -22,6 +22,10 @@
 from recommonmark.parser import CommonMarkParser
 
 
+# -- Project information -----------------------------------------------------
```

Comment:
nit: this change should be reverted! it will break vissl docs :)

Reply:
I think this still needs to be addressed?

Reply:
I think that came through the previous commits from cherry-picking.  It is fixed now.